serverless/tensorflow/faster_rcnn_inception_v2_coco/nuclio/main.py

```diff
@@ -11,7 +12,10 @@ def init_context(context):
     model_path = "/opt/nuclio/faster_rcnn/frozen_inference_graph.pb"
     model_handler = ModelLoader(model_path)
     setattr(context.user_data, 'model_handler', model_handler)
-    functionconfig = yaml.safe_load(open("/opt/nuclio/function.yaml"))
+    if tf.test.is_gpu_available() and tf.test.is_built_with_cuda():
```

Comment:
The TensorFlow documentation says that `is_gpu_available` is deprecated in favor of `tf.config.list_physical_devices('GPU')`. Also, the `is_built_with_cuda` check seems redundant - a GPU can't be available if there's no CUDA support.

Reply:
I understand that I don't need the `if` at all.